### PR TITLE
[15.0][FIX] account_invoice_date_due: Singleton error when user set date due on more than one invoice

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa TT42632

ping @carlosdauden @ernestotejeda

### CODE REVIEW

**Review by luisg123v — CHANGES_REQUESTED** [review]
Code LGTM, but regarding commit presentation:
- Missing description
- Remove `TT42632`, this gives no information to the community

**Review by carlosdauden — APPROVED** [review]

File: account_invoice_date_due/models/account_move.py [hunk 1]
```diff
@@ -37,10 +37,11 @@ def write(self, vals):
...
         # Propagate due date to move lines
         # that correspont to the receivable/payable account
-        if "invoice_date_due" in vals and self.state == "posted":
-            payment_term_lines = self.line_ids.filtered(
-                lambda line: line.account_id.user_type_id.type
-                in ("receivable", "payable")
-            )
-            payment_term_lines.write({"date_maturity": vals["invoice_date_due"]})
+        if "invoice_date_due" in vals:
+            for record in self.filtered(lambda r: r.state == "posted"):
```
Comment (luisg123v) [inline account_invoice_date_due/models/account_move.py]: Better filtering all records and the writting to all of them at once

**Review by luisg123v — CHANGES_REQUESTED** [review]

**Comment by sergio-teruel** [comment]
Changes done

**Review by pedrobaeza — APPROVED** [review]

**Review by ernestotejeda — APPROVED** [review]

**Pull request merged**